content/intro-to-reading-data.md

```diff
@@ -44,9 +51,9 @@ In this lesson we won’t consider much about accounts beyond their ability to s
 
 ### Public Keys
 
-Public keys are often referred to as addresses. The addresses point to accounts on the Solana network. If you want to run a specific program or transfer SOL, you’ll need to provide the necessary public key (or keys) to do so.
+Public keys are often referred to as addresses. The addresses point to accounts on the Solana network. If you want to run a specific program a.k.a smart contract or just want to transfer some SOL, you’ll need to provide the necessary public key (or keys) to do so.
 
-Public keys are 256-bit and they are often shown as base-58 encoded strings like `7C4jsPZpht42Tw6MjXWF56Q5RQUocjBBmciEjDa8HRtp`.
+Public keys are 32-bytes and they are often shown as [base-58](https://en.bitcoinwiki.org/wiki/Base58) encoded strings like `7C4jsPZpht42Tw6MjXWF56Q5RQUocjBBmciEjDa8HRtp`.
```

Comment:
Nice add 👍, most people have no idea what base58 is before they start hacking on blockchain.